feat/faction-chats

abdulqaharu opened the pull request:
<!-- enter the gh issue after hash -->

- [x] issue #106  
- [x] follows contribution [guide](https://github.com/keep-starknet-strange/art-peace/blob/main/CONTRIBUTING.md)

<!-- PR description below -->

b-j-roberts commented on backend/routes/quests.go:
```diff
@@ -260,3 +244,137 @@ func ClaimTodayQuestDevnet(w http.ResponseWriter, r *http.Request) {
+		canDelete = true
+	}
+
+	if !canDelete {
```
please merge any changes made into your other PR and fix all the changes mentioned there.

abdulqaharu commented:
When I updated the branch I was working off with latest code from main before I started working, it had so many conflicts and was a bit verbose to resolve. So i removed all my initial commit, updated the branch with the latest in main, before adding back my changes and resolved the comments you added and pushed. I am guessing doing that forced a push to the branch and closed this PR. 
So I had to created another PR.

@b-j-roberts
